langchain/chains/llm.py

```diff
@@ -184,11 +185,18 @@ async def aapply(
         await run_manager.on_chain_end({"outputs": outputs})
         return outputs
 
-    def create_outputs(self, response: LLMResult) -> List[Dict[str, str]]:
+    @property
+    def _run_output_key(self) -> str:
+        return self.output_key
+
+    def create_outputs(self, response: LLMResult) -> List[Dict[str, Any]]:
         """Create outputs from response."""
         return [
             # Get the text of the top generated string.
-            {self.output_key: generation[0].text}
+            {
+                self.output_key: self.output_parser.parse_result(generation),
+                "result": generation,
```

Comment:
Can this be a breaking change when used as input to another chain etc (because was 1 key before and now 2)

Reply:
i think only minorly

cause if they are using __call__ it returns a dictionary, so if they were doing something weird like `list(return)[0]` or something like that then ya. but if they were using `run` i changed it so i think it still works as intended, and i imagine most people either use run or use the dictionary as a whole

Reply:
actually thats wrong will mess things up with memory, will update to add a flag